src/main/java/org/damap/base/rest/invenio_damap/DMPPayload.java

```diff
@@ -0,0 +1,27 @@
+package org.damap.base.rest.invenio_damap;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.damap.base.rest.madmp.dto.Dataset;
+
+public class DMPPayload {
+  @JsonProperty("dmp_id")
+  private long dmp_id;
```

Comment:
```suggestion
  private long dmpId;
```
The json property annotation takes care of (de)serialization to/from `dmp_id`